src/dbus/server/dbus_thread_object.cpp

```diff
@@ -1394,6 +1394,10 @@ otError DBusThreadObject::GetDnssdCountersHandler(DBusMessageIter &aIter)
 
     dnssdCounters.mResolvedBySrp = otDnssdCounters.mResolvedBySrp;
 
```

Comment:
If you want to change the DBus interface, you need to modify the DBus interface definition at src/dbus/server/introspect.xml, DnssdCounters type. Double check if the client can talk correctly.
If you don't want to change the DBus interface, avoid the content serialization changes.

Reply:
I need to add three more metrics here, seems that I have to change the struct ?

    <!-- DnssdCounters: The DNS-SD counters
    <literallayout>
        struct {
          uint32 success
          uint32 server_failure
          uint32 format_error
          uint32 name_error
          uint32 not_implemented
          uint32 other
          uint32 resolved_by_srp

Reply:
I would suggest not do to so, or in separate PRs.

DBus clients will not work if this is changed unless the users updated their code.

Reply:
Thanks! Got it